src/API.jl

```diff
@@ -229,7 +229,9 @@ test(pkg::Union{AbstractString, PackageSpec}; kwargs...) = test([pkg]; kwargs...
 test(pkgs::Vector{<:AbstractString}; kwargs...)          = test([PackageSpec(pkg) for pkg in pkgs]; kwargs...)
 test(pkgs::Vector{PackageSpec}; kwargs...)               = test(Context(), pkgs; kwargs...)
 function test(ctx::Context, pkgs::Vector{PackageSpec};
-              coverage=false, test_fn=nothing, kwargs...)
+              coverage=false, test_fn=nothing,
+              julia_args::AbstractVector{<:AbstractString}=String[],
+              test_args::AbstractVector{<:AbstractString}=String[], kwargs...)
```

Comment:
I suggest to do something like
```
julia_args = String[a for a in julia_args]
test_args = String[a for a in test_args]
```
here at the entry point and use `Vector{String}` only for the internal methods. Or
```
julia_args = Cmd[a for a in julia_args]
```
if you go with the `Cmd` approach.

Reply:
Done.